Add guidance for making the switch to PostHog (#11195)
PostHog/posthog.com: contents/docs/new-to-posthog/switch-guide/switching-to-posthog.mdx

Reviewer: Overall, I don't think this is as effective as it should be. The content is great, but the format is letting it down a little. The "teams" sections should be:

- Intro paragraph
- 3-5 key points as bullets

That's it

Author: That should improve the skimmability, let me try that

---
title: "Switching to PostHog"
nextPage: ./planning-your-pilot.mdx
---

> For new projects and companies, PostHog is an easy sell: one login and one easy integration gets you everything you need to build great products.

Reviewer: Avoiding saying easy twice
```suggestion
For new projects and companies, PostHog is an easy sell: one login and simple integration gets you everything you need to build great products.
```

> But convincing your teammates to switch horses later in a product's life can require some planning and diplomacy. We made you a cheatsheet to help make the case.

Reviewer: What do you mean "later in a product's life". Our product (PostHog) or their product? I'm not sure it is the right phrasing.

Author: Maybe I don't need to say it all

> ## Sharing social proof

Reviewer: I think we should move this section down. Social proof is more supporting than the main reason someone will be convinced. Maybe the right place for this is at the bottom. Definitely worth including though.

> We’ve published [customer stories](/customers) with loads of companies who have loved their switch to PostHog. If you want to share a bit of social proof, these are a great place to start. Here are some choice quotes.

Reviewer: ```suggestion
We’ve published [customer stories](/customers) with loads of companies who have loved their switch to PostHog. Here are some choice quotes:
```

***

[ElevenLabs](/customers/elevenlabs):

<Quote
    imageSource="/images/customers/elevenlabs-sam.jpg"
    size="sm"
    name="Sam Sklar"
    title="Growth engineer, ElevenLabs"
    quote={`“We use basically everything in PostHog to understand more about our users and to plan new launches. We have an onboarding survey which is the first thing you see when you sign up.”`}
/>

***

[Octomind](/customers/octomind):

<Quote
    imageSource="/images/customers/maria_octomind.jpg"
    size="sm"
    name="Maria Zahorcova"
    title="Chief Marketing Officer, Octomind"
    quote={`“We’re product-led and it’s really, really important that we keep an eye not only on marketing activities but also the experience beyond that. It was great when I found out I could use PostHog to track data across the site, the docs, and the product — I just had to speak to our engineers about implementing it.”`}
/>

> ***

Reviewer: I think we should have one more quote here

## Advantages for everyone

PostHog is one login and one bill.

> That means less headache for more power, with all your data available across a range of products that help you understand user behavior as it happens.

Reviewer: This is a sentence fragment, feels like it is missing something

> PostHog is one login and one bill.
> 
> That means less headache for more power, with all your data available across a range of products that help you understand user behavior as it happens.

Reviewer: ```suggestion
PostHog is one login and one bill that means less headache for more power. All your data becomes available across a range of products to help you understand user behavior as it happens.
```

Author: I think if I say "this" instead of "that" it will parse better (first line really needs to stand alone), lmk how that lands

> You'll get a responsive team that cares about user needs, delivering improvements and customer requests constantly.

Reviewer: ```suggestion
You'll get a responsive team that cares about user needs, delivering improvements, and customer requests constantly.
```

> PostHog is completely agnostic to the language, framework and stack you build your product with. You can adopt it incrementally, adding more as needed.

Reviewer: ```suggestion
PostHog is completely agnostic to the language, framework, and stack you build your product with. You can adopt it incrementally, adding new functionality as needed.
```

> Best of all, PostHog is modern, fast, and built so that you’ll actually enjoy using it.

Reviewer: This is a weak point.

1. Honestly, we're not really that "fast", querying takes time and the interface can be complex.
2. I'm not sure what "modern" means.

Reviewer: Free to use and try might be a replacement if we haven't used that point too much.

Author: So by contrast, Amplitood, Muxpanel and GA are ANCIENT, right. We're the best balance of "sturdy and reliable" while built in this decade, and you can feel it in all the ways the product doesn't suck.

Some other way to say it maybe?

***

## Persuading product teams

> PostHog gives product teams the power to understand users, improve adoption and test assumptions. Your product teammates can get dive in immediately without having to wait for engineers to write new code or hook up an integration.

Reviewer: ```suggestion
PostHog gives product teams the power to understand users, improve adoption, and test assumptions. Your product teammates can dive in immediately without having to wait for engineers to write new code or hook up an integration.
```

> **[Report anything](/docs/product-analytics/insights)** with charts, trend reporting and [dashboards](/docs/product-analytics/dashboards) that are easy to build, but also deeply customizable for SQL enthusiasts.

Reviewer: ```suggestion
**[Visualize anything](/docs/product-analytics/insights)** with charts, trend reporting, and [dashboards](/docs/product-analytics/dashboards) that are easy to build, but also deeply customizable for SQL enthusiasts.
```

Author: nice

> **[Report anything](/docs/product-analytics/insights)** with charts, trend reporting and [dashboards](/docs/product-analytics/dashboards) that are easy to build, but also deeply customizable for SQL enthusiasts.

Reviewer: I think it makes sense to use a list for all of the below points

> **Learn anything you want** about everything the user touches in the product with [autocapture](/docs/product-analytics/autocapture).

Reviewer: ```suggestion
**Learn anything you want** about everything the user touches in the product without manual instrumentation with [autocapture](/docs/product-analytics/autocapture).
```

**Get clarity about navigation** and user priorities with [heatmaps](/docs/toolbar/heatmaps).

Use [session replay](/docs/session-replay/installation) to **reveal bugs, expose friction, and help instantly answer questions** like “is this new design working?”

> **Send events and other details to any other tool**, powering automations and keeping data in sync, with setup you can accomplish in seconds, fully customizable for any API or webhook. [Data pipelines](/docs/cdp) makes it easy.

Reviewer: I don't think this is a big selling point for product teams, but I could be wrong. Seems much more like a marketing thing...

Author: Apparently it comes up a lot, this was specifically suggested in the doc

> **Gather feedback** from exactly the users you want to reach with no-code [surveys](/docs/surveys/creating-surveys) that launch instantly.

Reviewer: ```suggestion
**Gather feedback and book user interviews** from exactly the users you want to reach with no-code [surveys](/docs/surveys/creating-surveys) that launch instantly.
```

> **Integrate revenue data** from [Stripe](/docs/cdp/sources/stripe) and other sources using data warehouse, so you can get a clear picture of what changes really move the needle.

Reviewer: I think this should be higher than sending events to other destinations

**Get unblocked by PostHog’s helpful sales and support teams,** ensuring you get the insights and information you need quickly.

***

## Good news for engineering teams

> PostHog is designed for developers. We’ve got a dedicated technical staff standing by to help keep you on track. When you reach out to support, you’ll be talking to someone who knows the technical details of the product really well.

Reviewer: I don't think "support" should be the first point. This one is better at the very least:

> The project is [developed in the open](https://github.com/PostHog/posthog), with a [public roadmap](/roadmap) and [dozens of pull requests landing]([https://github.com/PostHog/posthog/pulls?q=is%3Apr+is%3Aclosed](https://github.com/PostHog/posthog/pulls?q=sort%3Aupdated-desc+is%3Apr+is%3Aclosed)) each week.

While other products are low-velocity black boxes, we’re constantly making PostHog better.

The project is [developed in the open](https://github.com/PostHog/posthog), with a [public roadmap](/roadmap) and [dozens of pull requests landing](https://github.com/PostHog/posthog/pulls?q=is%3Apr+is%3Aclosed) each week.

Giving other teams the data they want becomes much easier with PostHog in the mix, with lower friction for engineering, and loads of features everyone can use without complex setup.

> **Consolidate the tools your company needs**, with one login, one integration, one bill, and one [deep set of docs](/docs).

Reviewer: Docs is sort of hidden in here, maybe its own point?

***

## Make the case to executives

We’re in this for a long-term relationship, so **PostHog wants you to spend as little money as possible**. Our sales team actually works with customers to reduce bills and optimize consumption. Instead of sneaky billing, we’d prefer to make more money as a result of you winning more business.

There’s just no sneaky stuff going on here. You can get started immediately with no shenanigans, no calls with the sales team.

- **Your team will like PostHog**. It’s fresh, always getting fixes, improvements and additional power. An investment in PostHog today means even more capabilities by next year.

- **There's no per-seat charge**, so PostHog makes it easy for everyone on your team to make data-driven decisions. No matter how quickly you scale your team, your costs remain fixed to the actual usage of your product. 

- **PostHog’s sales process is transparent and so is the whole company**: the company [handbook is public](/handbook), *even documenting the ways we can offer [discounts](/handbook/growth/sales/billing#coupons-and-discounts)*. Everything you want to know about pricing, processes and our product is publicly documented.

> - **You can have a free trial** to fully vet the costs and impact of your integration. Trying PostHog is low-risk, no cost, and fully reversible.

Author: EFFECTIVELY, everyone. If you're small you're covered under the free tier anyway, if you're huge, sales will just hook you up.

- **PostHog doesn’t require a contract**. You can save money with one, but it’s not a blocker to trying things out.

***

> ## Sharing social proof

Author: I'd like to keep it in the pitch guide to **hype the reader up** for the diplomacy they're about to do.

"No one ever got fired for picking Microsoft" kind of thing
